packages/plugin-ext/src/common/plugin-api-rpc.ts

```diff
@@ -991,7 +991,7 @@ export interface PreferenceRegistryExt {
 }
 
 export interface OutputChannelRegistryMain {
-    $append(channelName: string, value: string): PromiseLike<void>;
+    $append(channelName: string, value: string, pluginInfo: PluginInfo): PromiseLike<void>;
```

Comment:
In general, I have two comments:
1. should we make `pluginInfo` optional?
2. should it simply be in the form of `options? any` so we can potentially pass more in the future?

Reply:
Technically we could make it optional, but I was just following what I did in: https://github.com/eclipse-theia/theia/commit/f25652a25f47aaed68edf24560d96bed6d0e5fd4

Reply:
Okay fair enough :)